### PR TITLE
CLDC-2672 Update save and cancel buttons for duplicates

### PR DESCRIPTION
**Pull request by kosiakkatrina**
Instead of having `Save and continue` and `Skip for now` buttons when updating duplicate logs, we want it to act similar to check your answers and have `Save changes` and `Cancel`

### CODE REVIEW

File: app/views/form/page.html.erb [hunk 1]
```diff
@@ -63,12 +63,15 @@
       <%= f.hidden_field :interruption_page_referrer_type, value: @interruption_page_referrer_type %>
 
       <div class="govuk-button-group">
-        <% if !@page.interruption_screen? && if request.query_parameters["referrer"] != "check_answers" %>
+        <% if !@page.interruption_screen? && if request.query_parameters["referrer"] != "check_answers" && request.query_parameters["referrer"] != "duplicate_logs" %>
```
Comment (natdeanlewissoftwire) [inline app/views/form/page.html.erb]: if this is merged after https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/1983 then it should also exclude the `duplicate_logs_banner` referrer as it stands

File: app/views/form/page.html.erb [hunk 2]
```diff
@@ -63,12 +63,15 @@
...
           <%= f.govuk_submit "Save and continue" %>
           <%= govuk_link_to(
             (@page.skip_text || "Skip for now"),
             (@page.skip_href(@log) || send(@log.form.next_page_redirect_path(@page, @log, current_user), @log)),
           ) %>
+          <% elsif request.query_parameters["referrer"] == "duplicate_logs" %>
```
Comment (natdeanlewissoftwire) [inline app/views/form/page.html.erb]: similarly this would be `duplicate_logs` OR `duplicate_logs_banner`

**Review by natdeanlewissoftwire — APPROVED** [review]
lgtm!

File: app/views/form/page.html.erb (hunk 1 above)
Comment (kosiakkatrina) [reply app/views/form/page.html.erb]: Yup, just saw that!

Comment (natdeanlewissoftwire) [reply app/views/form/page.html.erb]: I've just merged the above PR so this extra referrer check will need to be in this PR please :) happy to pair if you like/want any clarification

**Pull request merged**